proto/lucidity/lucidity.proto

```diff
@@ -8,12 +8,16 @@ option go_package = "github.com/thought-machine/please-servers/proto/lucidity";
 // central server and send some basic information about their status.
 service Lucidity {
   // Update sends the server an update on the current state of a worker.
-  rpc Update(UpdateRequest) returns (UpdateResponse);
+  rpc Update(Worker) returns (UpdateResponse);
+  // List the current set of workers
+  rpc ListWorkers(ListWorkersRequest) returns (ListWorkersResponse);
 }
 
-message UpdateRequest {
+message Worker {
```

Comment:
This adds quite a bit of noise to the PR but it seemed much nicer to name it this way, especially now there > 1 RPC in this file